torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2155,6 +2155,18 @@ def sample_inputs_max_min_binary(op_info, device, dtype, requires_grad, **kwargs
                   for input_tensor, other_tensor in args_for_binary_op)
     return inputs
 
+def sample_inputs_dropout(self, device, dtype, requires_grad):
+    tensors = []
+    dropout_args = [
```

Comment:
It might be simpler to pair this with another sequence of shapes, like `((), (5,) , (5, 5), (5, 5, 5))`